lib/raft/raft_impl.h

```diff
@@ -338,8 +338,8 @@ template <typename Server> class RaftImpl : public Raft<Server>
     if (!i_am_leader())
       return;
     // Attempt to pass leadership to a worthy successor.
-    const ::std::string *best_node = nullptr;
-    NodeState *best                = nullptr;
+    const ::std::string *best_node = NULL;
+    NodeState *best                = NULL;
```

Comment:
Yeah, but I figured just to be complete. Perhaps @SolidWallOfCode or @zwoop has a different opinion.